README.md

```diff
@@ -3,8 +3,44 @@
 If you are the kind of person who likes thinking _long-term_ and _high-impact_
 this repo is for you!
 
-**D**o **W**hat **Y**ou **L**ove is _not_ a 3-month stepping-stone project ...
+**D**o **W**hat **Y**ou **L**ove is _not_ a stepping-stone project, we're in it for the long-haul. <br />
```

Comment:
The reason I commented that is because I was under the impression that part of the vision is no longer treating dwyl as an acronym?